documentation/staging/content/userguide/managing-domains/domain-events.md

```diff
@@ -285,3 +288,100 @@ Source:
 Type:    Normal
 Events:  <none>
 ```
+
+Example of the sequence of operator generated events in a domain rolling restart after the domain resource's `image` and `logHomeEnabled` changed.
+
+The output of command `kubectl get events -n sample-domain1-ns --selector=weblogic.domainUID=sample-domain1,weblogic.createdByOperator=true --sort-by=lastTimestamp'.
```

Comment:
output of command -> output of the command
Instead of a period at the end of this phrase (it is not not a complete sentence), it should be a colon.